tasks/es/cnig.py

```diff
@@ -157,16 +158,18 @@ class Geometry(TableTask):
 
     resolution = Parameter()
     timestamp = Parameter(default='20150101')
+    id_aux = Parameter()
```

Comment:
Instead of creating a new table with the Canary Islands, the Canary Islands should be added to the existing table.

Essentially, the `TableTask` should require two different `ImportGeometry` tasks, and then run `INSERT` once for each of them.